jamato14/smart-mirror
# Weather API

**What does this PR do:**

This adds the weather component to the website. It uses WeatherAPI.com to query data then displays it on the homepage. Eventually this module may include multiple options of apis to query weather data from.

**How to test:**
npm start from home directory
verify that the weather shows up in the top right hand corner
